### PR TITLE
Reported speech

### PR DESCRIPTION
**Pull request by nschneid**
New policy for reported speech (much simpler to explain!).

### CODE REVIEW

File: _u-dep/ccomp.md [hunk 1]
```diff
@@ -41,6 +41,62 @@ sentence to mean that the boss will not be doing any digging, in the second
...
+
+Quoted content is considered to be [ccomp]() even if it is a sentence fragment:
+
+~~~ sdparse
+" Three/NUM muffins/NOUN , " he answered .
+nummod(muffins, Three)
+ccomp(answered, muffins)
+~~~
+
+If the speech verb interrupts the reported speech content, [parataxis]() is used instead:
```
Comment (amir-zeldes) [inline _u-dep/ccomp.md]: Even though it's evident from the example, I would add: "parataxis is used instead *and attached to the root of the reported speech, in this example the word "all"*.

**Comment by amir-zeldes** [comment]
LGTM, added just one tiny comment

**Comment by nschneid** [comment]
@dan-zeman do you want to take a look or should I go ahead and merge?

**Comment by dan-zeman** [comment]
> @dan-zeman do you want to take a look or should I go ahead and merge?

I took a quick look just now; didn't spot anything wrong.

**Comment by agu-oli** [comment]
Does this policy applied to all languages or only to Latin, Italian and Sicilian?

**Pull request merged**